test/big_query/impressions_test.exs

```diff
@@ -6,7 +6,7 @@ defmodule Castle.BigQueryImpressionsTest do
   import BigQuery.Impressions
 
```

Comment:
oops, the last test in this set is failing as well (3600 instead of BigQuery.TimestampRollups.Hourly or such)

Reply:
Now how did i miss that?  Will fix!